q

```diff
@@ -0,0 +1,230 @@
+
+                   SSUUMMMMAARRYY OOFF LLEESSSS CCOOMMMMAANNDDSS
```

Comment:
Spectacular fails! wtf?

I *did* see that there were properly only 3 files. And i most certainly didn't notice *that one* anywhere. The only thing that comes to mind is that i messed up squashing my commits. One line had "quash" and so git rejected it, told me to do `git rebase --edit-todo` then `--continue`. But i don't remember `less` being anywhere in the picture. Besides, i know how to use less.

lol I'll see if i can figure it out at this end.